### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by jchetboun** [facebookresearch/denoiser#24]
Hi!

First, thank you for the awesome work!

I was interested in getting more details about how you trained your SOTA model. If I understand correctly, you first trained the model on the Valentini dataset, but how did you partition the data into train, validation and test set (there is no such partition when I download it)? Then you used the best model from this first training as a restart for the training with the DNS dataset, right? Did you then use both of the datasets together or only DNS in this second stage? Also for DNS, how did you build the noisy audio samples and how did you partition into train and test set?

Thank you in advance!

**Comment by adiyoss** [comment]
Hi,
You can download valentini from this link: https://datashare.is.ed.ac.uk/handle/10283/2791 (here there is a split between train and test). For validation, we sampled mixture data from two other speakers in the training set.
Regarding DNS, you can get the noisy files from their repo: https://github.com/microsoft/DNS-Challenge (make sure you are on the interspeech 2020 branch). For test files we used the ones provided by the authors (also in this repo).  
We used both datasets together with WHAM (http://wham.whisper.ai/) in the master model only.

**Comment by jchetboun** [comment]
Thanks a lot! And what about the training strategy? Did you train with the 3 datasets together at one time or did you first trained only on Valentini then fine tuned on DNS and WHAM? In the paper you suggest different losses and data augmentation depending on the dataset.

**Comment by adiyoss** [comment]
In the paper we trained an evaluated on each dataset separately, meaning training on DNS and evaluating on DNS, etc. 
Some of the pre-trained models we provide (i.e., the master models) were trained on all the datasets jointly. 
For the master model we used bandmask, shift, and revecho augmentations, and optimize both L1 and STFT loss functions

**Comment by jchetboun** [comment]
Thank you for the details!

**Comment by adiyoss** [comment]
Closing this issue, in case you have other issues reopen it.
